src/test/scala/firrtlTests/MemEnFeedbackSpec.scala

```diff
@@ -0,0 +1,41 @@
+// See LICENSE for license details.
```

Comment:
Should this go into a "regressions" package?
In general this mostly seem to be testing that the combinatorial loop bug is fixed.
It is not a generic test of sync-read memories.

Reply:
I could see arguments both ways (it is testing a corner case), but this type of test is currently interspersed among many of the other test files. Finding and moving them might make sense in a later PR.